### ISSUE TITLE
aurman parses incorrectly pacman.conf and takes comments in account

### ISSUE THREAD

**Issue by ochurlaud** [polygamma/aurman#216]
## Description

In my /etc/pacman.conf I had: 

```
IgnorePkg   = ttf-win7-fonts  # kdevelop kdevelop-pg-qt amarok
```

Upon initialization aurman would do: 
```
~~ initializing aurman...
:: Ignoring installed package kdevelop-pg-qt
:: Ignoring installed package ttf-win7-fonts
:: Ignoring installed package kdevelop
```

## Expected Behavior

aurman should not do anything particular about commented packages

## Current Behavior

Upon initialization aurman currently does (where kdeveop-pg-qt and kdevelop are commenterd packages): 
```
~~ initializing aurman...
:: Ignoring installed package kdevelop-pg-qt
:: Ignoring installed package ttf-win7-fonts
:: Ignoring installed package kdevelop
```

## Version of aurman you are using

```
:: 2.16.9-1.
```
## Steps to Reproduce

1. In /etc/pacman.conf do : `IgnorePkg   = ttf-win7-fonts  # kdevelop kdevelop-pg-qt amarok`
2. Run aurman

## Read the README

Yes

## Running linux distribution

Archlinux

> **Notice**: If you do not fill out one of the fields, without mentioning **why** you left it out, the issue will be closed without any comment.
You may also be banned from this repository without further warning.

**Comment by eli-schwartz** [comment]
aurman does not parse pacman.conf, it lets pyalpm do the parsing. And pyalpm uses pacman's own shared library for most of the work.

Also, pacman.conf has never documented its support for comments and the ability to start them in the middle of a line -- it worked anyway, true, but then this was removed in https://bugs.archlinux.org/task/48702 for some pretty good reasons.

tl;dr pacman itself does the same, therefore aurman's behavior here is correct.

**Comment by polygamma** [comment]
see: https://github.com/polygamma/aurman/commit/21927f83397fb1db541252842f9f07cb7147224b

**Comment by ochurlaud** [comment]
@polygamma I understand the closing with the given rationale.

Your last comment doesn't bring anything. I thought it should have worked this way. I was wrong and this happens. No need to be harsh about it.

**Comment by polygamma** [comment]
Sorry, should not be harsh. But there are many "bug reports" lately, which are invalid due to the fact that the wanted expected behavior is wrong, hence I decided to append that to the template. I posted the commit here, because I finally decided to do so after reading your bug report
